tests/functional/watcherdecisionengine_controller_test.go

```diff
@@ -145,14 +181,41 @@ var _ = Describe("WatcherDecisionEngine controller", func() {
 				corev1.ConditionTrue,
 			)
 		})
-		It("should have ReadyCondition ready", func() {
+		It("should have config service input ready", func() {
```

Comment:
don't we already have this test defined at line 168?

Reply:
yes my bad removing duplicate

Reply:
not sure if Github UI is confusing me but it looks to me like this test case is a duplicate of the one above

Reply:
yes, there is a duplicated test It("should have config service input ready",

Reply:
yes sorry :/ removing